build.gradle.kts

```diff
@@ -32,6 +32,8 @@ dependencies {
     api("io.micrometer:micrometer-core:1.3.3")
     api("com.newrelic.telemetry:telemetry:0.6.1")
     implementation("org.slf4j:slf4j-api:1.7.30")
+    implementation("ch.qos.logback:logback-classic:1.2.3")
+    implementation("ch.qos.logback:logback-core:1.2.3")
```

Comment:
Why?  Last I remembered, logback is dual licensed with LGPL included.  Do we really want/need to do this?

Reply:
This is a snyk test. I'm trying to figure out why it's failing even though the issue is ignored in Snyk.

Reply:
I want to see the Snyk check fail _with_ these libs and pass _without_ them.